I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 285, in script
    show BLACK_SCREEN with fade(5.0)
TypeError: 'MultipleTransition' object is not callable

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 285, in script
    show BLACK_SCREEN with fade(5.0)
  File "D:\VPR24\VPR24\YA GAY\renpy\ast.py", line 1678, in execute
    renpy.exports.with_statement(trans, paired=paired)
  File "D:\VPR24\VPR24\YA GAY\renpy\exports.py", line 1758, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "D:\VPR24\VPR24\YA GAY\renpy\display\core.py", line 2978, in do_with
    return self.interact(trans_pause=True,
  File "D:\VPR24\VPR24\YA GAY\renpy\display\core.py", line 3582, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "D:\VPR24\VPR24\YA GAY\renpy\display\core.py", line 3976, in interact_core
    trans = instantiate_transition(None, old_root, layers_root)
  File "D:\VPR24\VPR24\YA GAY\renpy\display\core.py", line 3922, in instantiate_transition
    trans = self.ongoing_transition[layer](
TypeError: 'MultipleTransition' object is not callable

Windows-10-10.0.22621 AMD64
Ren'Py 8.1.3.23091805
VPR24 1.0
Wed Oct 18 18:29:32 2023
